packages/circuit-ui/hooks/useFocusList/useFocusList.docs.mdx

```diff
@@ -0,0 +1,20 @@
+import { Status, Story } from '../../../../.storybook/components';
+
+# useFocusList()
+
+<Status.Stable />
+
+Enables keyboard navigation for a list of focusable elements.
+
+<Story id="hooks-usefocuslist--example" />
+
+```ts
+useFocusList(): {
+  'data-focus-list': string;
+  'onKeyDown': (event: KeyboardEvent) => void;
+};
+```
+
+## Usage
+
+Application menus such as popover menus, custom select dropdowns, and navigation lists should be [navigatable by keyboard](https://inclusive-components.design/menus-menu-buttons/#keyboardandfocusbehavior). This hook enables users to navigate between menu items using the arrow up/down keys. It does not manage focus when opening/closing a menu, make sure to implement this as well.
```

Comment:
- I think that this also applies to radio button or checkbox groups, we could mention them alongside the custom selects
- > It does not manage focus when opening/closing a menu, make sure to implement this as well.
  
  Can we add an example of this to the Story?

Reply:
1. Radio button and checkbox groups do this natively, so this hook isn't needed for them. 
2. I feel like that's a bit beyond the scope of these docs, especially because it depends on the context. 🤔 I'll link to the Popover implementation as an example.

Reply:
Done in 126cdfffa.